yarn-project/txe/src/txe_service/txe_service.ts

```diff
@@ -123,6 +123,11 @@ export class TXEService {
 
   // PXE oracles
 
+  utilityAssertOracleVersionMatches(version: ForeignCallSingle) {
+    this.executor.utilityAssertOracleVersionMatches(fromSingle(version).toNumber());
+    return toForeignCallResult([]);
```

Comment:
```suggestion
  utilityAssertOracleVersionMatches(foreignVersion: ForeignCallSingle) {
    const version = fromSingle(version).toNumber();
    
    this.executor.utilityAssertOracleVersionMatches(version);
    
    return toForeignCallResult([]);
```

This is the code style introduced in https://github.com/AztecProtocol/aztec-packages/pull/16537